services/app-web/src/pages/StateSubmission/ReviewSubmit/V2/ReviewSubmit/RateDetailsSummarySectionV2.tsx

```diff
@@ -77,8 +78,8 @@ export const RateDetailsSummarySectionV2 = ({
         contract.draftRevision?.formData ||
         getLastContractSubmission(contract)?.contractRevision.formData
     const rates =
-        contract.draftRates ||
```

Comment:
little weird to me that we're just grabbing one or the other. 

I think we should be pulling from one or the other depending on isEditing. if isEditing then we are reading from draftRates, if not, then the latest submitted rates

Reply:
@macrael I _believe_ it's because of types that we do this.Everything is optional, so you want to narrow it early to avoid that optionality messing up the UI.  If you do the current implementation of `isEditing` then either value could still be undefined which trickles down. 

Def think we could use type assertions or more precise tools eventually. Maybe we need a design ticket on that? However, right now coming out GraphQL we  don't have a union type that' s either a draft or a locked package, you have a type two optional subfields that determine what is being used, so you need to clear that early.

Reply:
WIth this change, getting draftRates after checking for last submission, as a state user you won't see the current version of the rates if you are editing a submission that has been submitted before. Looking at it again I think this is a bug